frontend/src/metabase/collections/components/BulkActions.jsx

```diff
@@ -67,27 +33,66 @@ function BulkActions(props) {
     isNavbarOpen,
   } = props;
 
-  const canMove = selected.every(item => canMoveItem(item, collection));
-  const canArchive = selected.every(item => canArchiveItem(item, collection));
+  const onMoveClick = selected.every(item => canMoveItem(item, collection))
+    ? onMoveStart
+    : null;
+  const onArchiveClick = selected.every(item =>
+    canArchiveItem(item, collection),
+  )
+    ? onArchive
+    : null;
+
+  const showing = selected.length > 0;
 
   return (
-    <BulkActionBar showing={selected.length > 0} isNavbarOpen={isNavbarOpen}>
```

Comment:
To update the UI to match the new design for bulk actions, I decided to no longer use `BulkActionBar` here and instead just add the UI directly to this component.

Updating `BulkActionBar` with the new UI wasn't feasible since that component is used somewhere else in the app.

I also chose not to make a new component for the sake of simplicity, it would be more complex and convoluted to split the UI and styles across two different components given how lightweight it is.